source/hardware/controllers/numark_dj2go2_touch.rst

```diff
@@ -0,0 +1,28 @@
+.. _numark-dj2go2-touch:
+
+Numark DJ2GO2 Touch
+===================
+
+.. sectionauthor::
+   tandy1000 <https://github.com/tandy-1000>
+
+The Numark DJ2GO2 Touch is a USB :term:`MIDI` controller with an integrated audio interface.
+It is the successor to the DJ2GO2, and features touch capable wheels.
+
+-  `Manufacturer’s product page <https://www.numark.com/product/dj2go2-touch>`__
+-  `Serato DJ Hardware Page <https://serato.com/dj/hardware/numark-dj2go2-touch>`__
+-  `Mapping Forum Thread <https://mixxx.discourse.group/t/a-complete-numark-dj2go2-touch-mapping/22616>`__
+
+.. versionadded:: 2.3.1
+
+Controller Mapping
+------------------
+
+Everything in this mapping is standard except:
+ - the Level knobs have been remapped to a filter effect,
+ - the browse encoder can be used to preview tracks, with an added seek functionality.
```

Comment:
Please be explicit. This is not a changelog, its a manual entry. Telling the user what features exist doesn't make much sense if they have no Idea how to access it. Describe how exactly this feature can be used on the controller.

Reply:
good point, I just committed an improvement.

Reply:
Thanks, much clearer now.